lib/hubspot/connection.rb

```diff
@@ -37,7 +38,7 @@ def put_json(path, options)
 
       def delete_json(path, opts)
         url = generate_url(path, opts)
-        response = delete(url, format: :json)
+        response = delete(url, format: :json, hubspot: Hubspot::Config.timeout)
```

Comment:
This parameter should be `timeout` not `hubspot`

Reply:
fixed